.github/scripts/draft-change-log-entries.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+previous_version=$1
+
+range="$previous_version..HEAD"
+
+echo "## Unreleased"
+echo
+
+git log --reverse \
+        --perl-regexp \
+        --author='^(?!renovate\[bot\] )' \
+        --pretty=format:"* %s" \
+        "$range" \
+  | sed -E 's,\(#([0-9]+)\)$,\n  ([#\1](https://github.com/open-telemetry/opentelemetry-configuration/pull/\1)),'
```

Comment:
This is a script we use in `opentelemetry-java` to hep prepare uniform changelog entries